src/transformers/__init__.py

```diff
@@ -849,6 +855,17 @@
 
     # PyTorch models structure
 
+    _import_structure["models.vivit"].extend(
+        [
+            "VIVIT_PRETRAINED_MODEL_ARCHIVE_LIST",
+            "ViViTLayer",
```

Comment:
You can remove ViViTLayer and load_tf_weights_in_vivit everywhere from the inits